index.html

```diff
@@ -70,222 +80,244 @@ <h1>I’m Alexandra. Glad to see you!</h1>
+                <br>
+                <div class="flex-item-width">
+                    <button id="seeProject-button">See Project</button>
+                </div>
             </div>
         </div>
         <!-- Work section item four -->
-        <div class="works-grid-item">
-            <div class="template-img flex-item-width">
-                <img src="img/Snapshoot Portfolio 2.png" alt="">
-            </div>
-            <div class="Tonic flex-item-width">
-                <h2>Multi-Post Stories
-                </h2>
-            </div>
-            <div class="ol-list flex-item-width">
-                <ul>
-                    <li id="canopy">CANOPY</li>
-                    <li>Back End Dev</li>
-                    <li>2015</li>
-                </ul>
-            </div>
-            <div class="flex-item-width paragraph">
-                <p>
-                    A daily selection of privately personalized reads; no accounts or sign-ups required.
-                </p>
+        <div class="works-grid-item item-desktop-2">
+            <div class="desktop">
+                <div class="template-img flex-item-width">
+                    <img src="img/Snapshoot Portfolio 2.png" alt="">
+                </div>
             </div>
-            <div class="flex-item-width buttons-flix">
-                <ul class="ul-list">
-                    <button class="buttons"> <li>HTML</li></button>
-                    <button class="buttons"><li>CSS</li></button>
-                    <button class="buttons">
+            <div class="desktop">
+                <div class="Tonic flex-item-width">
+                    <h2>Multi-Post Stories
+                    </h2>
+                </div>
+                <div class="ol-list flex-item-width">
+                    <ul>
+                        <li id="canopy">CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="flex-item-width paragraph">
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="flex-item-width buttons-flix">
+                    <ul class="ul-list">
+                        <button class="buttons"> <li>HTML</li></button>
+                        <button class="buttons"><li>CSS</li></button>
+                        <button class="buttons">
             <li>JavaScript</li></button>
-                </ul>
-            </div>
-            <br>
-            <div class="flex-item-width">
-                <button id="seeProject-button">See Project</button>
+                    </ul>
+                </div>
+                <br>
+                <div class="flex-item-width">
+                    <button id="seeProject-button">See Project</button>
+                </div>
             </div>
```

Comment:
- [ Required ] I think it would be better if the project details in your project cards are aligned just as they appear in the Figma design. In the Figma design, the _Project Title_  has a small `margin` from the top compared to the `margin` on your design which appears to be a bit bigger. Remember that you want to showcase your design skills and bringing out designs just as they are in design documents is very key. Please see the screenshot below and rectify this.

## Your design
![Screenshot from 2023-05-04 22-54-28](https://user-images.githubusercontent.com/16392046/236316435-31524942-7a66-447b-bc95-197a7ae45c50.png)

## Figma design 
![Screenshot from 2023-05-04 22-54-04](https://user-images.githubusercontent.com/16392046/236316743-6c5f468d-4a81-47af-b0d7-f6d9f0d49bd9.png)